src/components/Plants/PlantingSiteDetails.tsx

```diff
@@ -89,12 +95,14 @@ export default function PlantingSiteDetails(props: PlantingSiteDetailsProps): JS
     };
 
     populateResults();
-  }, [selectedPlantingSite]);
+  }, [plantingSite]);
 
   return (
     <Grid container display='flex' marginTop={6}>
-      <PlantingSiteMap plots={plots} />
-      <Grid item xs={isMobile ? 12 : 6} sx={{ paddingLeft: 1 }}>
+      <Grid item xs={isMobile ? 12 : 6} sx={mapCardStyle}>
+        <PlantingSiteDashboardMap plots={plots} siteId={plantingSite?.id} />
```

Comment:
Why don't we pass the plantingSite object instead of the id, so we don't need to fetch the planting site again in the PlantingSiteDashboardMap component?

Reply:
yes, good point. not sure why i did it this way.

Reply:
oh wait, yes, we don't get the polygon info here

Reply:
that's why fetching full details of the planting site

Reply:
We need to fetch the planting site here to get the full details of zones and plots with polygons.
The planting site information at the parent level is shallow (no zones/plots) and we should keep it that way for performance.

Reply:
Oh, you mean the planting zones?

Reply:
Yes the zones and plots with polygon info. To show on map

Reply:
When we list all planting sites in the parent we can theoretically fetch all the info for every planting site. That’s one option but it will be expensive.